redbot/setup.py

```diff
@@ -251,7 +251,7 @@ async def remove_instance(
 
     if interactive is True and delete_data is None:
         delete_data = click.confirm(
-            "Would you like to delete this instance's data?", default=False
+            "Would you like to delete this instance's data? If you have not created the database, please select No.", default=False
```

Comment:
This should only be shown when a non-JSON backend is used. Also, you should just say that the removal of instance's data requires the database server to be running as what you currently put is not the only case this can happen.

Reply:
Not sure how to check which backend is in use, would I get it from this?
Line 176 `default_dirs["STORAGE_TYPE"] = storage_type.value`

Reply:
There's a `backend = get_current_backend(instance)` line a bit below, just move it a bit higher and then you can check it with `if backend != BackendType.JSON`

Reply:
I think I did that right - it's still failing style though. Not sure why